static/js/src/cube/header-app.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { QueryClient, QueryClientProvider } from "react-query";
+import CubeHeader from "./components/CubeHeader";
+import useMicrocertsData from "./hooks/useMicrocertsData";
+
+const oneHour = 1000 * 60 * 60;
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+      refetchOnReconnect: false,
+      staleTime: oneHour,
+      retryOnMount: false,
+    },
+  },
+});
+
+const App = () => {
+  const {
+    modules,
+    certifiedBadge,
+    hasEnrollments,
+    isLoading,
+  } = useMicrocertsData();
+
+  return (
+    <CubeHeader
+      modules={modules}
+      certifiedBadge={certifiedBadge}
+      hasEnrollments={hasEnrollments}
+      isLoading={isLoading}
+    />
+  );
+};
+
+const HeaderApp = () => {
+  return (
+    <QueryClientProvider client={queryClient}>
+      <App />
+    </QueryClientProvider>
+  );
+};
+
+export default HeaderApp;
```

Comment:
Wait, are we creating a new App just for the header? Couldn't this live in the same one as the microcerts table?

Reply:
The template has different divs where the header and table are rendered, which is why they're two separate apps right now. When we reactify the rest of the page in https://github.com/canonical-web-and-design/commercial-squad/issues/424 everything can be under a single app.